x-pack/plugins/security_solution/public/detections/components/endpoint_responder/use_responder_action_item.tsx

```diff
@@ -39,10 +41,18 @@ export const useResponderActionItem = (
     [eventDetailsData]
   );
 
+  const { agentStatus } = useHostIsolationStatus({ agentId: endpointId });
```

Comment:
This is redundant because `<ResponderContextMenuItem>` already pulls in the agent's metadata document and checks status.
(also - this hook seems to be doing much more than just getting agent status, so its a little overkill to be used here)